parsers/jscript.c

```diff
@@ -922,6 +918,16 @@ static void parseRegExp (void)
 /* Read a C identifier beginning with "first_char" and places it into
  * "name".
  */
+
+static bool include_period_in_identifier = false;
```

Comment:
How about making this an integer?

If this variable is used in a nested context (unexpectedly), debugging related to this variable may be hard.

If we use int for the variable, we may need the following functions.

```
static int include_period_in_identifier = 0;

static void accept_period_in_identifier()
{
    include_period_in_identifier++;
}

static void reject_period_in_identifier()
{
    include_period_in_identifier--;
}
```

Just an idea.

Reply:
Another idea is adding `include_period_in_identifier` field to tokenInfo.
The code can be;

```
token.include_period_in_identifier++;
readToken(token);
token.include_period_in_identifier--;
```

Reply:
Switched to integer.